pkg/descheduler/apis/config/v1alpha2/types_pluginargs.go

```diff
@@ -143,9 +145,9 @@ type MigrationControllerArgs struct {
 	DefaultJobTTL *metav1.Duration `json:"defaultJobTTL,omitempty"`
 
 	// EvictQPS controls the number of evict per second
-	EvictQPS string `json:"evictQPS,omitempty"`
+	EvictQPS *config.Float64OrString `json:"evictQPS,omitempty"`
```

Comment:
tha's very thoughtful, but is it really necessary?

Reply:
Yes..Users naturally think that the evictQPS field needs to be filled with numbers and needs to be of floating point type. In order to be compatible with stock scenarios, it is now necessary to support both floating point and string types

Reply:
I see, that makes sense to me.